src/glossary.tsx

```diff
@@ -0,0 +1,280 @@
+import { Disclosure } from '@headlessui/react';
+import clsx from 'clsx';
+import FocusTrap from 'focus-trap-react';
+import { useEffect, useId, useRef } from 'react';
+import { lock, unlock } from 'tua-body-scroll-lock';
+import { IncentiveType } from './api/calculator-types-v1';
+import { useTranslated } from './i18n/use-translated';
+import { CircledChevron, Cross } from './icons';
+
+/** A single accordion section. */
+function Section({
+  title,
+  paragraphs,
+  defaultExpanded,
+}: {
+  title: string;
+  paragraphs: string[];
+  defaultExpanded?: boolean;
+}) {
+  return (
+    <Disclosure
+      as="div"
+      defaultOpen={defaultExpanded}
+      className={clsx(
+        'group',
+        'flex',
+        'flex-col',
+        'py-4',
+        'gap-4',
+        'border-b',
+        'last:border-b-0',
+        'border-color-border-secondary',
+      )}
+    >
+      <Disclosure.Button
+        autoFocus={defaultExpanded}
+        className="flex items-center text-left"
+      >
+        <h2 className="grow text-lg font-medium leading-tight">{title}</h2>
+        <span className="group-data-open:rotate-180">
+          <CircledChevron
+            w={40}
+            h={40}
+            fillClass="fill-[#ebebeb] group-data-open:fill-color-action-primary"
+            strokeClass="stroke-grey-900 group-data-open:stroke-color-text-primary-on-dark"
+          />
+        </span>
+      </Disclosure.Button>
+      <Disclosure.Panel className="flex flex-col gap-4">
```

Comment:
Re: VoiceOver - Should the focus go to the button or go to the panel with the content? I'm not the best with VoiceOver, but I have used it before and I was having some issues getting VoiceOver to read the content as it was just iterating through the buttons.

I think there is some controversy over setting tabindex to do this, but that is possibly one way? https://webaim.org/techniques/keyboard/tabindex

But here is the applicable WCAG standard https://www.w3.org/WAI/WCAG21/Understanding/focus-order.html

Reply:
It works for me in my main browser (Brave; Chromium-based). But I wouldn't be surprised if other browsers behave differently; I'm consistently surprised at how much variety there is in how browsers build the accessibility tree.

Also, the content is just plain `<p>`s with text inside; I can't imagine any browser would ignore those?